Packs/DB2/Integrations/DB2/README.md

```diff
@@ -0,0 +1,53 @@
+Integration to provide connectivity to IBM DB2 using the python ibm_db2 library.
+This integration was integrated and tested with version 0.1 of DB2
+
+## Configure DB2 on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for DB2.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Database host | True |
+    | Port | False |
+    | Database Name | False |
+    | Username | True |
+    | Password | True |
+    | Connection Arguments (ex: arg1=val1&amp;arg2=val2) | False |
+    | Use an SSL connection | False |
+    | Use Persistent Connection | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### db2-query
+***
+Running a DB2 query command
+
+
+#### Base Command
+
+`db2-query`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| query | The DB2 query to run. | Required | 
+| limit | The maximum number of results. Default is 50. | Optional | 
+| skip | The offset at which to start the result. Default is 0. | Optional | 
+| bind_variables_name | A comma separated list of names which will be replaced in query having ':&lt;name&gt;'. | Optional | 
+| bind_variables_values | A comma separated list of values corresponding to `bind_variables_name` or values in order of '?' mark to be replaced in query. | Optional | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
```

Comment:
better to give an example here, but not a must

Reply:
added in [3fe28ed](https://github.com/demisto/content/pull/18139/commits/3fe28edaae14abdcd24bc1f718d86478d6bfd3ab)